s4e-backend/src/main/java/pl/cyfronet/s4e/controller/response/BasicProductResponse.java

```diff
@@ -1,14 +1,16 @@
 package pl.cyfronet.s4e.controller.response;
 
 import org.springframework.beans.factory.annotation.Value;
+import pl.cyfronet.s4e.bean.ProductCategory;
 
 public interface BasicProductResponse {
     Long getId();
-
     String getName();
-
     String getDisplayName();
 
     @Value("#{@productService.isFavourite(target.id)}")
-    boolean isFavourite();
-}
+    boolean getFavourite();
```

Comment:
I'd leave the old name, i.e. isFavourite here, as it's a boolean.

Reply:
It's the Java problem with casting names with the prefix `is` and using `@Data` `@Builder`. I don't know why, but filed name `isFavourite` in response is called `favourite`. It's propagated trough whole UI since the implementation of `favourites`. 

To keep old implementation state in UI I used field `getFavourite` to be mapped correctly with old state of returned response (in interfaces prefix `is` is correctly cast).